src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/InternalCalls.cs

```diff
@@ -158,8 +156,8 @@ internal static int RhEndNoGCRegion()
         [RuntimeImport(Redhawk.BaseName, "RhpGcSafeZeroMemory")]
         internal static extern unsafe ref byte RhpGcSafeZeroMemory(ref byte dmem, nuint size);
 
-        [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        [RuntimeImport(Redhawk.BaseName, "memmove")]
+        [DllImport(Redhawk.BaseName)]
+        [UnmanagedCallConv(CallConvs = [typeof(CallConvCdecl)])]
```

Comment:
```suggestion
        [MethodImplAttribute(MethodImplOptions.InternalCall)]
        [RuntimeImport(Redhawk.BaseName, "memmove")]
```
This is the only outstanding issue in this PR that can be handled in follow up. The rest LGTM